builder/role_image_test.go

```diff
@@ -101,8 +101,6 @@ func TestGenerateRoleImageRunScript(t *testing.T) {
 
 	runScriptContents, err := roleImageBuilder.generateRunScript(rolesManifest.Roles[0])
 	assert.NoError(err)
-	assert.Contains(string(runScriptContents), "/var/vcap/jobs-src/tor/templates/data/properties.sh.erb")
```

Comment:
These are now referred to in `TestGenerateRoleImageJobsConfig()` because they're in the jobs json file now.